docs/docs/faq.md

```diff
@@ -5,57 +5,19 @@
 Nebari regenerates this file on every run. This means it will be removed by the operating system during its cleanup process,
 but running the `nebari deploy` command again as a Nebari administrator will update/create a `NEBARI_KUBECONFIG` file for you.
 
-## How are Nebari conda user environments created? Who creates them?
+## How are conda user environments created? Who creates them?
 
-The short answer: there are currently _two_ ways of creating environments, as we are in the process of migrating Nebari to conda-store,
-and so which way depends on your use-case.
+`Conda-store` manages all environments on the Nebari. It allows users to create private environments in their own namespace, or shared environments under a group namespace.
 
-The longer answer:
+Additionally, there is a legacy approach which is still available in Nebari. Administrators can create global environments by specifying the environment in `nebari-config.yml`. Environments specified in this way will be made available for all users and services under the `nebari-git` namespace on `conda-store`. We do not recommend this approach as it requires a redeploy in order to update the environment, though it may be useful in some circumstances.
 
-- For global environments, you can specify the environment in `nebari-config.yml`, and it will be made available for all users and services.
-- By comparison, creating the environments through conda-store will provide more granular control over certain settings and permissions.
-
-As Nebari and conda-store mature, the intent is to migrate exclusively to conda-store for environment creation and management.
```

Comment:
Side comment, but conda store is more mature now.  Maybe we should open an issue to deprecate it.